README.md

```diff
@@ -0,0 +1,156 @@
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 HakcSummit 2023 <a name="about-project"></a>
+
+**HackSummit 2023** is a capstone project for HTML CSS and JavaScript module at microverse
+
+## 🛠 Built With <a name="built-with"> HTML CSS & Java Script</a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> HTML CSS & Java Script
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **Presents the Home page for HackSummit 2023**
+- **Presents the About page for HackSummit 2023**
+- **Displays Speakers List with more and less button functionality**
+- **Displays Mobile Menu**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> This projcet is not deployed currently
+
+- [Live Demo Link]()
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need a web browser
+
+### Setup
+
+Clone this repository to your desired folder:
+
+### Install
+
+Install this project using the cloned repo or download and use the files to your local machine.
+
+### Usage
+
+To run the project open the index.html file in your web browser
+
+### Run tests
+
+No test are available
+
+### Deployment
+
+This project is not currently deployed
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Kaleb Bekele**
+
+- GitHub: [@githubhandle](https://github.com/kalbek)
+- Twitter: [@twitterhandle](https://twitter.com/KNuramo)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/kaleb-nuramo/)
+- Website: https://kalbek.github.io/Portfolio/
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **More Content will be personalized**
+- [ ] **Other pages will be created and linked to the home page**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] Kindly, Provide valid links in your README File, to fully present your Project professionally. For this issue page, you may use this[ link ](https://github.com/kalbek/Capstone-1-Hack-Summit/issues)instead 🙏